README.md

```diff
@@ -24,6 +24,23 @@ One could also use a standard Clang installation, build a sysroot from the
 sources mentioned above, and compile with
 "--target=wasm32-wasi --sysroot=/path/to/sysroot".
 
+## Install
+
+A typical installation from the release binaries might look like the following:
+```shell script
+wget https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-[VERSION]/wasi-sdk-[VERSION]-linux.tar.gz
```

Comment:
Do you know what we need to do to update this URL to point to the WebAssembly org now, rather than CraneStation?

Reply:
Huh, looks I used an URL from somewhere; the new ones work too. I'll update.